src/SolutionTemplate/UnoSolutionTemplate/Wasm/UnoQuickStart.Wasm.csproj

```diff
@@ -35,18 +35,17 @@
 
     You can safely remove this ItemGroup completely.
     -->
-		<Compile Remove="Program.cs" />
-		<Compile Include="Program.cs" />
-		<Content Include="LinkerConfig.xml" />
-		<Content Include="wwwroot\web.config" />
+		<None Include="Program.cs" />
```

Comment:
Should this be `Compile`?

Reply:
It's automatically compile as it is in the folder. It has to be included in the project for the template engine to recognize that the file needs to be copied over... so `None` it is.